exercises/09_dedup/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func Dedup(s string) string {
+	var outputr []rune
+	var prev rune
```

Comment:
it's a good try! but it wouldn't work if the string started with ascii 0s

Reply:
![image](https://user-images.githubusercontent.com/65840/147661093-cdd12596-2e65-40a2-ab44-08f47efdc6de.png)